lec12-datasets.Rmd

```diff
@@ -0,0 +1,159 @@
+    HypothesisFalse [label = "Hypothesis\nif FALSE", pos = "1.5,1.5!"]
+    Report [label = "Report/Publish", pos = "0.75,0.75!"]
+
+    Hypothesis -> CollectData -> AnalyzeData -> HypothesisTrue -> Report
+    AnalyzeData -> HypothesisFalse -> {Report Hypothesis}
+}')
+```
+
+### Steps in the scientific process[^sci_method]
+
+1. Identify research question(s).
+2. Look into what the previous literature shows. <!--(how much emphasis on this?)-->
+3. Create one or more hypotheses or objectives.
+4. Write up an outline or expected approach to answering those questions/objectives (analysis and presentation plan):
+    - How will the data be obtained and what is the data (i.e. the source).
+    - What statistical or mathematical techniques have previous researchers used? Will they be used or will other ones be used? Are they appropriate (optional, may need an expert)? 
+    - How will the results/data be presented or visualized?
+    - Identify/describe some possible tables and figures to present the results/data.
+5. Run all analyses (or additional ones that may come up as you work with and analyze the data).
+6. Visualize or present all results from the analyses.
+7. Interpret the results and how they fit in the context of the previous literature and findings.
+8. Draw conclusions based on the hypotheses/objectives.
+9. Publish your results (in blogs, pre-print archives, general archiving servers, journals, or at conferences)
+
+
+[^sci_method]: See [Khan Academy on Scientific Method](https://www.khanacademy.org/science/biology/intro-to-biology/science-of-biology/a/the-science-of-biology) for a brief overview and explanation on the scientific process.
+
+## Team dynamics
+
+### Basics of succeeding with a team
+
+Final assignment is in a group and to succeed, you need to understand team dynamics:
+
+- Everyone is different, every team is different
+- Communication is vital to work together and to achieve the goal
+- Teams go through various stages
+- Need consensus for group norms, goals, duties/responsibilities, and conduct/behaviour
+    - Important that everyone has a stake in the project
+- Rotate roles (specifically for the leader/facilitator)
+
+### Stages of group formation
+
+- "Forming-Storming-Norming-Performing-Adjourning Model"[^group_forming]
+- Essentially, groups go through stages (not always all and not always in order):
+    1. Getting to know one other
+    2. Feeling comfortable and safer, testing boundaries
+    3. Opening up, voicing opinions, potential for conflict
+    4. Stronger bonds form, cooperative
+    5. Greater focus and energy on completion of project
+    6. Team ends after project is successfully completed
+- These stages can be fast-tracked by discussing norms, duties, and conduct early on
+
+[^group_forming]: See [Principles of Group Dynamics](https://open.lib.umn.edu/principlesmanagement/chapter/13-3-group-dynamics/) 
+
+### Roles and responsibilities
+
```

Comment:
Any idea on how to divide them into these roles if we do 4 ppl per project? Any comments on the grouping of people I did in the google sheets btw?

Reply:
I saw further down that it is up to the students to decide this good.